src/Modules/OrchardCore.Commerce.Inventory/Drivers/InventoryPartDisplayDriver.cs

```diff
@@ -36,7 +38,7 @@ public class InventoryPartDisplayDriver : ContentPartDisplayDriver<InventoryPart
             var currentSku = _hca.HttpContext?.Request.Form["ProductPart.Sku"].ToString().ToUpperInvariant();
             var skuBefore = viewModel.Inventory.FirstOrDefault().Key != null
                 ? viewModel.Inventory.FirstOrDefault().Key.Split('-')[0]
-                : "DEFAULT";
+                : currentSku;
```

Comment:
Shouldn't `DefaultKey` should be used here? (Or the const should be removed) Because the const now is not used anywhere.